src/lib/zombienetInfo.ts

```diff
@@ -1,44 +1,44 @@
-export type zombienetInfo = typeof zombienetBins;
+export type zombienetInfo = typeof zombienet;
 
 export type BinaryNames = "zombienet" | "polkadot" | "astar-collator";
-export const zombienetBins = {
-  "zombienet": {
-    version: "1.3.89",
-    downloadUrl: {
-      darwin: {
-        "arm64": "https://github.com/paritytech/zombienet/releases/download/v1.3.89/zombienet-macos",
-        "x64": "https://github.com/paritytech/zombienet/releases/download/v1.3.89/zombienet-macos"
-      },
-      linux: {
-        "arm64": "https://github.com/paritytech/zombienet/releases/download/v1.3.89/zombienet-linux-arm64",
-        "x64": "https://github.com/paritytech/zombienet/releases/download/v1.3.89/zombienet-linux-x64",
-      }
+export const zombienet= {
+  version: "1.3.89",
+  downloadUrl: {
+    darwin: {
+      "arm64": "https://github.com/paritytech/zombienet/releases/download/v${version}/zombienet-macos",
+      "x64": "https://github.com/paritytech/zombienet/releases/download/v${version}/zombienet-macos"
     },
-  },
-  "polkadot": {
-    version: "0.9.43",
-    downloadUrl: {
-      darwin: {
-        "arm64": "https://github.com/paritytech/polkadot/releases/download/v0.9.43/polkadot",
-        "x64": "https://github.com/paritytech/polkadot/releases/download/v0.9.43/polkadot",
-      },
-      linux: {
-        "arm64": "https://github.com/paritytech/polkadot/releases/download/v0.9.43/polkadot",
-        "x64": "https://github.com/paritytech/polkadot/releases/download/v0.9.43/polkadot",
-      }
+    linux: {
+      "arm64": "https://github.com/paritytech/zombienet/releases/download/v${version}/zombienet-linux-arm64",
+      "x64": "https://github.com/paritytech/zombienet/releases/download/v${version}/zombienet-linux-x64",
     }
   },
-  "astar-collator": {
-    version: "5.28.0",
-    downloadUrl: {
-      darwin: {
-        "arm64": "https://github.com/AstarNetwork/Astar/releases/download/v5.28.0/astar-collator-v5.28.0-macOS-x86_64.tar.gz",
-        "x64": "https://github.com/AstarNetwork/Astar/releases/download/v5.28.0/astar-collator-v5.28.0-macOS-x86_64.tar.gz",
+  binaries: {
+    "polkadot": {
+      version: "0.9.43",
+      downloadUrl: {
+        darwin: {
+          "arm64": "https://github.com/paritytech/polkadot/releases/download/v${version}/polkadot",
```

Comment:
This is the old repo, the new one is [polkadot-sdk](https://github.com/paritytech/polkadot-sdk/releases).
You can add `polkadot-parachain` binary from the same one.

Reply:
The problem is that `astar-collator` suports [Polkadot v0.9.43](https://github.com/paritytech/polkadot/releases/tag/v0.9.43)

Reply:
Also, I have problem to launch zombienet with Polkadot >=v1.1.0. I have this errors:
```bash
╔════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ All relay chain nodes spawned...                                                                                       ║
╚════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
(node:15873) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `zombienet --trace-warnings ...` to show where the warning was created)

Error    fetching metrics from: http://127.0.0.1:38919/metrics

Error    fetching metrics from: http://127.0.0.1:40677/metrics

Error    fetching metrics from: http://127.0.0.1:44201/metrics

Error    fetching metrics from: http://127.0.0.1:38919/metrics

Error    fetching metrics from: http://127.0.0.1:40677/metrics
```

Reply:
@prxgr4mm3r I just [spotted](https://github.com/inkdevhub/swanky-node/blob/489d12e4245f831bbe160c47c71afdefa5f99e06/runtime/Cargo.toml#L42-L44) that `swanky-node` uses `polkadot-v0.9.39` versions.
Let's stick to `polkadot-v0.9.43` version for all: `polkadot`, `polkadot-parachain` & `astar-collator`.
The update will be made in another issue.